src/transport/raw/PeerAddress.h

```diff
@@ -208,6 +216,9 @@ class PeerAddress
     {
         return TCP(addr).SetPort(port).SetInterface(interface);
     }
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF
+    static PeerAddress WiFiPAF() { return PeerAddress(Type::kWiFiPAF); }
```

Comment:
A PeerAddress of type WiFiPAF should actually carry the relevant details like the peer MAC address, or at least some sort of identifier to distinguish connections at the lower level.

Reply:
I will add the identifier